### DISCUSSION TITLE
confused, does this use fuse3 in raspberry pi debian bookworm ?

### DISCUSSION THREAD

**Discussion by hydra3333** [trapexit/mergerfs#1348] [Q&A]
Hello, I have a raspberry pi 5, can mergerfs use fuse3 via apt install (or download and install the newest) or do i need to install the old fuse2 ?  Thanks

**Comment by trapexit** [comment]
https://github.com/trapexit/mergerfs?tab=readme-ov-file#is-my-oss-libfuse-needed-for-mergerfs-to-work

mergerfs does not use the system libfuse.

**Reply by hydra3333** [reply]
Beaut, thank you.
